salvo/README.md

```diff
@@ -1,12 +1,35 @@
 # Salvo
 
+## What is Salvo
+
 This is a framework that abstracts executing multiple benchmarks of the Envoy Proxy using [NightHawk](https://github.com/envoyproxy/nighthawk).
 
-## Example Control Documents
+## Goals of Salvo
+
+Salvo allows Envoy developers to perform A/B testing to monitor performance change of Envoy. Salvo provides the local excution mode allowing developers to run a benchark on their own machine and also provides the remote excution mode to run a benchmark on a remote machine, such as a remote CI system.
+
+## Dependencies
+
+The `install_deps.sh` script can be used to install any dependencies required by Salvo.
+
+## Building Salvo
+
+To build Salvo, use the following command:
+
+```bash
+bazel build //...
+```
+
+## Benchmark Test Cases for Salvo
+
+Benchmark test cases for Salvo are defined as Python files with test cases written in pytest framework, here is an example: https://github.com/envoyproxy/nighthawk/blob/main/benchmarks/test/test_discovery.py. Users can provide thier own Python files of test cases into Savlo.
+
+## Control Documents
 
-The control document defines the data needed to execute a benchmark. At the moment, we support the fully dockerized benchmark and the scavenging benchmark.  The work for the binary benchmark is in progress.
+The control document defines the data needed to execute a benchmark. At the moment, we support the fully dockerized benchmark and the scavenging benchmark. The work for the binary benchmark is in progress.
```

Comment:
I was under the impression that the binary benchmark was also implemented. See:

- https://github.com/envoyproxy/envoy-perf/pull/113
- https://github.com/envoyproxy/envoy-perf/pull/115

This seems important, since the binary benchmark is likely going to be the one most useful for Envoy developers.

Reply:
Thank you for the information, fixed.

Reply:
> This seems important, since the binary benchmark is likely going to be the one most useful for Envoy developers.

@mum4k I'm drafting the doc about the workflow of Envoy deveploers. In my opinion, as a Envoy developer, the binary benchmark is the fastest way to test his/her own codes changes whithout building a image. Is this reason the same as yours? Can you leave your thoughts on it and about why not use the Dockerized/Scavenging Benchmark? Thanks